components/formats-gpl/src/loci/formats/in/InveonReader.java

```diff
@@ -110,7 +110,7 @@ public boolean isThisType(RandomAccessInputStream stream) throws IOException {
   {
     FormatTools.checkPlaneParameters(this, no, buf.length, x, y, w, h);
 
-    int planeSize = FormatTools.getPlaneSize(this);
+    long planeSize = (long) FormatTools.getPlaneSize(this);
```

Comment:
Yes, it's certainly outside the scope of this PR, just thought it worth mentioning if it's a more general limitation we might want to address as a follow-on task.